ls1intum/Artemis
# `Development`: Update GitLab and Jenkins setup in development documentation

### Checklist
#### General



### Motivation and Context


The Screenshots and some commands/navigation hints were outdated. I had to setup GitLab and Jenkins recently and thought I'd help the next person to do so.
### Description

Just updating the documentation to keep it accurate.

### Steps for Testing

https://artemis-platform--7145.org.readthedocs.build/en/7145/dev/setup.html#jenkins-and-gitlab-setup
Look at your GitLab/Jenkins Setup and see that the new screenshots are correct, the process only changes for the creation of the administrator user, as it is not possible anymore to create a password upon user creation.

### Review Progress






#### Code Review
#### Manual Tests
#### Exam Mode Test
